docs/settings/triggers/snowboy.md

```diff
@@ -49,3 +70,27 @@ Then, open an issue or create a pull request to add the model to the list bellow
 | [kalliope-RU](https://snowboy.kitt.ai/hotword/2964)  | Russian  | каллиопа     |
 | [kalliope-DE](https://snowboy.kitt.ai/hotword/4324)  | German   | Ka-lio-pe    |
 | [kalliope-IT](https://snowboy.kitt.ai/hotword/10650) | Italian  | Ka-lljo-pe   |
+
+
+
+## Pretrained universal models
+
+Snowboy provides pretrained universal models.
+Ŷou can either find them [here](https://github.com/Kitt-AI/snowboy/tree/master/resources) or in the kalliope repo under `kalliope/kalliope/trigger/snowboy/resources/models/umdl` 
+
+Here is the list of the umdl models, and the parameters that you have to use for them:
+
+* alexa.umdl: Universal model for the hotword "Alexa". Set `apply_frontend` to `True`.
```

Comment:
I would prefer to leave to user download his own model.
Furthermore we plan to remove snowboy so if we could avoid sending binary for a temp thing...